pkg/dispatchcli/cmd/update.go

```diff
@@ -27,20 +28,16 @@ func NewCmdUpdate(out io.Writer, errOut io.Writer) *cobra.Command {
 		Long:    updateLong,
 		Example: updateExample,
 		Run: func(cmd *cobra.Command, args []string) {
-			if file == "" {
-				runHelp(cmd, args)
-				return
-			}
-
-			updateMap := map[string]modelAction{
-				"secret": CallUpdateSecret,
-			}
-
-			err := importFile(out, errOut, cmd, args, updateMap)
+			err := runUpdate(out, errOut, cmd, args)
```

Comment:
I had originally left this in because I was ignorant of what it was doing for the create methods. Currently update works by taking in an entity name and a path to the json you want to update. So `dispatch update base-image nodejs6-base update.json`. This other support was meant to allow users to do `dispatch update -f update.json` (borrowed from the original `dispatch create -f seed.yaml`. If that is still desired, I will make sure it still works properly.

Reply:
I don't think it hurts anything to keep it. Certainly since we support it on create and delete.

Reply:
I'm making sure to add this for both the base-image and api update commands.